modules/order/src/Entity/Order.php

```diff
@@ -389,6 +406,14 @@ public function setData($data) {
   /**
    * {@inheritdoc}
    */
+  public function addAdjustment(Adjustment $adjustment) {
+    $this->get('adjustments')->appendItem($adjustment);
+    return $this;
```

Comment:
Indeed. We also want getAdjustments() / setAdjustments / removeAdjustment, right?